h/static/styles/threads.scss

```diff
@@ -17,7 +17,9 @@ $threadexp-width: .6em;
 }
 
 .thread-list {
-  margin-top: 0.5em;
+  .thread {
+    margin-top: 0.5em;
+  }
```

Comment:
This adds margin to every `thread` in the list now, perhaps you meant to just add the margin to the `first-child`?